ui/draggable.js

```diff
@@ -12,6 +12,8 @@
 //>>description: Enables dragging functionality for any element.
 //>>docs: http://api.jqueryui.com/draggable/
 //>>demos: http://jqueryui.com/draggable/
+//>>css.structure: ../themes/base/draggable.css
+//>>css.theme: ../themes/base/theme.css
```

Comment:
@scottgonzalez can you confirm this, and for other interactions except resizable?